app/police-custody-form/index.js

```diff
@@ -6,7 +6,14 @@ const policeCustodyFormController = require('./controllers')
 // Define routes
 router.get('/', (req, res) => {
   delete res.breadcrumb
+  res.locals.formErrors = req.session.errors
+  res.locals.showErrorsSummary = req.session.showErrorsSummary
+  res.locals.formData = req.session.formData
+
   res.render('police-custody-form/police-custody-form')
+  delete req.session.errors
+  delete req.session.showErrorsSummary
+  delete req.session.formData
```

Comment:
I'm wondering why we need to delete these?

Reply:
We delete these as if you fill the form and you have errors ect, if you leave that page and come back it will maintain the errors, error summary and the form data. Not sure we wanted that to be kept after leaving the page and coming back